packages/webapp/src/components/SensorReadingsLineChart/styles.module.scss

```diff
@@ -1,44 +1,43 @@
-.lineChartWrapper{
+.lineChartWrapper {
   max-height: 650px;
 }
 .title {
   font-style: normal;
   font-weight: normal;
-  color: var(--green);
-  font-size: 20px;
+  font-size: 24px;
```

Comment:
Maybe this should be added to the title for empty chart? (`packages/webapp/src/containers/SensorReadingsLineChart/index.jsx` new line 50)
![Screenshot 2023-05-23 at 1 00 14 PM](https://github.com/LiteFarmOrg/LiteFarm/assets/33141219/335fc840-8c96-4a27-820a-aa4f5a66c578)

Reply:
Excellent eye as always!